package.json

```diff
@@ -53,10 +53,11 @@
       "node": true
     },
     "extends": "eslint:recommended",
-    "parser": "babel-eslint",
+    "parser": "@babel/eslint-parser",
     "parserOptions": {
       "ecmaVersion": 2018,
       "impliedStrict": true,
+      "requireConfigFile": false,
```

Comment:
Explanation of this field: https://github.com/babel/babel/tree/main/eslint/babel-eslint-parser#additional-parser-configuration